app-misc/task/task-2.6.0~git.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="Short description of ProjectX"
+DESCRIPTION="Long ProjectX description.
+Spanning several paragraphs, maybe
+ - containing
+ - bullet points etc.
+You can also have really long lines that do not fit in 80 characters and use \
+a backslash to wrap the line."
+HOMEPAGE="https://homepage/of/projectx.org"
+COPYRIGHT="2014 Developer name
+	2016 Another developer's name"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="73575f8d88dfc95198ae4a5bd31a2f7f0c74251b"
+SOURCE_URI="https://github.com/GothenburgBitFactory/taskwarrior/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="3cace85261bd5ff822baa5eeb315435896c7f7fe998b34fb20c10cc40f214973"
+srcGitRev_2="9a5f24e2acb38d05afb8f8e316a966dee196a42a"
+SOURCE_URI_2="https://github.com/GothenburgBitFactory/libshared/archive/$srcGitRev_2.tar.gz"
+CHECKSUM_SHA256_2="2abc6e862c83f12138e1d736f13f503b726587eae350dd42594c7ad83d7463bf"
+SOURCE_FILENAME="task-v$portVersion-$srcGitRev.tar.gz"
+SOURCE_FILENAME_2="libshared-$srcGitRev_2.tar.gz"
+SOURCE_DIR="taskwarrior-$srcGitRev"
+PATCHES="task-$portVersion.patchset"
+PATCHES_2="task-$portVersion-source2.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	task$secondaryArchSuffix = $portVersion
+	cmd:task = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgnutls$secondaryArchSuffix
+	lib:libuuid$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libgnutls$secondaryArchSuffix
+	devel:libuuid$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+PATCH()
+{
+	sed -i '/scripts/d' CMakeLists.txt
+}
+
+TEST_REQUIRES="
+	cmd:python3
+	"
+
+BUILD()
+{
+	cp -rf $sourceDir2/libshared-$srcGitRev_2/* $sourceDir/src/libshared
+
+	mkdir -p build && cd build
+	cmake -DCMAKE_BUILD_TYPE=Release \
+		-DTASK_BINDIR=$prefix/bin \
+		-DTASK_DOCDIR=$docDir/ \
+		-DTASK_MAN1DIR=$manDir/man1 \
+		-DTASK_MAN5DIR=$manDir/man5 \
+		-DTASK_RCDIR=$docDir/rc \
+		-DBUILD_SHARED_LIBS=off \
+		-DCMAKE_EXE_LINKER_FLAGS=-lnetwork ..
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make -C build install
+}
+
+TEST()
+{
+	# produces a few crashes, 2712 tests OK, 1 test FAILED
+	cd build/test && make $jobArgs && ./run_all -v && cat all.log | grep 'not ok' && ./problems
```

Comment:
This line is too long

Reply:
Yeah ;) checked on how other OS's mentioned  it in the docker files, will reduce to the standard in a bit :)
One patch is up streamed also, so that can be removed too, working on it

Reply:
Fixed :)